examples/sql_bank/main.go

```diff
@@ -49,10 +52,15 @@ func moveMoney(db *sql.DB) {
 		if err != nil {
 			log.Fatal(err)
 		}
-		if _, err = tx.Exec("SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
-			log.Fatal(err)
-		}
+		startTime := time.Now()
+		// TODO(vivek): Query is very slow, needs investigation.
```

Comment:
Great. BTW your suggestion didn't work. I've added a test to demonstrate it. You can modify the output of those tests once you've added the point support.